metrology/reporter/statsd.py

```diff
@@ -0,0 +1,112 @@
+import re
+import socket
+import sys
+
+from metrology.instruments import *  # noqa
+from metrology.reporter.base import Reporter
+
+
+class StatsDReporter(Reporter):
+    """
+    A statsd reporter that send metrics to statsd daemon ::
+
+      reporter = StatsDReporter('statsd.local', 2003)
```

Comment:
wouldn't a default 'localhost' be better? and the default statsd port is AFAIR 8125